includes/plugins/class-newspack-newsletters.php

```diff
@@ -73,7 +73,7 @@ public static function newspack_newsletters_contact_data( $provider, $contact, $
 						if ( is_wp_error( $existing_contact ) ) {
 							Logger::log( 'Adding metadata to a new contact.' );
 							$is_new_contact = true;
-							if ( empty( $selected_list_ids ) ) {
+							if ( false === $selected_list_ids || empty( $selected_list_ids ) ) {
```

Comment:
`true === empty( false )`, so no need for the extra check here

Reply:
Done in f8ca0b4dc8fb47c8c1533e997046b7940b2bd8b8